day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,42 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping
+    puts "Here are your current toppings: #{toppings}"
+    puts "Do you want to add a topping?"
+    new_topping = gets.chomp
```

Comment:
I really like this set up, but if I answered the question "Do you want to add a new topping?" With the response "No", that would get tossed into your toppings array! Easy fix- just change what you ask, but thought I'd call attention to this because these are exactly the types of edge cases developers are on the look out for!